editor/src/clj/editor/gui.clj

```diff
@@ -425,6 +425,17 @@
         (g/set-property source :child-index next-index)
         (attach-gui-node node-tree target source type)))))
 
+(defn- gen-outline-node-tx-attach-fn
```

Comment:
Should we make this public and mark it `;; SDK api` so we can also make the corresponding changes to the `SpineScenesNode` in `extension-spine`?

https://github.com/defold/extension-spine/blob/main/defold-spine/editor/src/spineguiext.clj#L309

Reply:
Good idea! I did that 👍